lib/inky.rb

```diff
@@ -58,7 +58,7 @@ def transform_doc(elem)
     def self.extract_raws(string)
       raws = []
       i = 0
-      regex = %r(< *raw *>(.*?)</ *raw *>)
+      regex = %r(<\s*raw\s*>((?!</\s*raw\s*>).*?)</\s*raw\s*>)misux
```

Comment:
Aren't the `x` and `u` flag superfluous here? The `s` flag is either superfluous or wrong too.

Reply:
this was the combination of flag I had to use since ruby regex doesn't allow dynamic negative lookahead. I tried it with multiple combinations and this was the one that didn't skip closing tags when there was a second raw tag to match

Reply:
the `s` flag is so `.*?` includes linebreaks it's definitely needed

Reply:
i think you're right though, `x` is not needed, I wasn't sure about the unicode flag `u`, so I included it, will remove those and see if tests still pass

Reply:
I guess ruby's regex parser is a little different than the flags I expect to be needed, you are correct about `x` and `s`, but ` u` is still necessary otherwise there are encoding errors. especially since a lot of people code with UTF-8 on by default

Reply:
squashed and re-pushed